pkg/apis/environment/basic_view.go

```diff
@@ -65,9 +68,46 @@ type (
 		runtime.RequestCollection[
 			predicate.Environment, environment.OrderOption,
 		] `query:",inline"`
+
+		IncludeSummary bool `query:"includeSummary,omitempty"`
 	}
 
-	CollectionGetResponse = []*model.EnvironmentOutput
+	CollectionGetResponse = []*environmentOutput
 )
 
 type CollectionDeleteRequest = model.EnvironmentDeleteInputs
+
+type environmentOutput struct {
+	model.EnvironmentOutput `json:",inline"`
+	StatusSummary           status.Count `json:"statusSummary"`
+}
+
+func exposeEnvironment(entity *model.Environment) *environmentOutput {
```

Comment:
Can we move it to pkg/dao/expose.go

Reply:
`func exposeEnvironment` only changes the expose API response object, and no other places need to use this function, prefer to keep it in the API level, like other expose functions (exposeSetting, exposeVariable)